src/Swarm/TUI/View/Achievement.hs

```diff
@@ -0,0 +1,84 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Swarm.TUI.View.Achievement where
+
+import Brick
+import Brick.Widgets.Border (borderWithLabel)
+import Brick.Widgets.Center (hCenter)
+import Brick.Widgets.List qualified as BL
+import Control.Lens ((^.))
+import Data.Map (Map)
+import Data.Map qualified as M
+import Swarm.TUI.Attr
+import Swarm.TUI.Model
+import Swarm.TUI.Model.Achievement.Attainment
+import Swarm.TUI.Model.Achievement.Definitions
+import Swarm.TUI.Model.Achievement.Description
+import Swarm.TUI.Model.UI
+
+padAllEvenly :: Int -> Widget Name -> Widget Name
+padAllEvenly x w = padTopBottom x $ padLeftRight (2 * x) w
+
+getCompletionIcon :: Bool -> Widget Name
+getCompletionIcon = \case
+  False -> txt " ○  "
+  True -> withAttr greenAttr $ txt " ●  "
+
+drawAchievementsMenuUI :: AppState -> BL.List Name CategorizedAchievement -> Widget Name
+drawAchievementsMenuUI s l =
+  vBox
+    [ hCenter $ padTopBottom 1 $ str "🏆  Achievements 🏆 "
+    , hBox
+        [ hLimitPercent 50 $
```

Comment:
This wastes a lot of space and doesn't look very good on bigger terminals:
![achievements-menu](https://user-images.githubusercontent.com/533859/207953687-4dfc166f-4c85-4fbc-99bf-364360e59ca0.png)

Perhaps there should be some proportional padding to the left and right so the menu and achievement details don't extend all the way to the edges of the window?

Reply:
Here's the new look:

| Narrow | Wide |
| --- | --- |
| ![Screenshot from 2022-12-17 14-53-55](https://user-images.githubusercontent.com/261693/208268916-4c2fec0d-0700-46f7-97a2-6617ca921fe5.png) | ![Screenshot from 2022-12-17 14-49-16](https://user-images.githubusercontent.com/261693/208268925-1f9d747a-06a1-4f11-982e-8ece688a27f0.png) |